src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -1373,8 +1380,9 @@ export class SavedObjectsRepository {
       .update<unknown, unknown, SavedObjectsRawDocSource>({
         id: this._serializer.generateRawId(namespace, type, id),
         index: this.getIndexForType(type),
-        ...getExpectedVersionProperties(version, preflightResult?.rawDocSource),
+        ...getExpectedVersionProperties(version),
```

Comment:
@jportner we already discuss about it a while ago, but can you confirm this is fine with you?

Reply:
Yes, I think we don't have to use OCC for the preflight check.

However, if you're removing the "preflight OCC" here, WDYT about doing it in the other functions too for consistency? (`bulkCreate`, `bulkUpdate`, `delete`)

Reply:
Yea, I agree, we should do it, and we should do it in the current PR ihmo.